enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -775,7 +775,25 @@ To answer this question, we need to do the following:
 * Verify that OpenShift Service Mesh will include support for this feature in time for this EP.
 * Evaluate any potential security concerns around this feature.
 
-**Resolution**: TBD.
+**Resolution**: Yes. However, whether ReferenceGrants will be functional, is dependent on the
```

Comment:
I would have answered No.  If they aren't functional, that's automatically a No to me.